nwb_conversion_tools/nwbconverter.py

```diff
@@ -59,28 +63,28 @@ def get_metadata(self):
             metadata = dict_deep_update(metadata, interface_metadata)
         return metadata
 
-    def run_conversion(self, metadata_dict, nwbfile_path=None, save_to_file=True, conversion_options=None):
+    def run_conversion(self, metadata, nwbfile_path=None, save_to_file=True, conversion_options=None):
         """Build nwbfile object, auto-populate with minimal values if missing."""
         if conversion_options is None:
             conversion_options = dict()
         nwbfile_kwargs = dict()
-        if 'NWBFile' in metadata_dict:
-            nwbfile_kwargs.update(metadata_dict['NWBFile'])
-        if 'Subject' in metadata_dict:
-            nwbfile_kwargs.update(subject=Subject(**metadata_dict['Subject']))
+        if 'NWBFile' in metadata:
+            nwbfile_kwargs.update(metadata['NWBFile'])
+        if 'Subject' in metadata:
+            nwbfile_kwargs.update(subject=Subject(**metadata['Subject']))
         # convert ISO 8601 string to datetime
         if isinstance(nwbfile_kwargs['session_start_time'], str):
             nwbfile_kwargs['session_start_time'] = datetime.fromisoformat(
-                metadata_dict['NWBFile']['session_start_time']
+                metadata['NWBFile']['session_start_time']
             )
         nwbfile = NWBFile(**nwbfile_kwargs)
 
         for interface_name, data_interface in self.data_interface_objects.items():
-            these_conversion_options = get_schema_data(
-                conversion_options,
+            valid_conversion_options = get_schema_data(
+                conversion_options[interface_name],
```

Comment:
This is the only line that actually changes the functionality (and as a consequence bumps this up to v0.6.0). All downstream dependents will not have to pass conversion options on a per interface basis.

Reply:
@CodyCBakerPhD can you check the changes I proposed on 96fed83 and 1c7eec5? I think we won't need the `get_schema_data` step, since we should validate the data against schema for source, conversion_options and metadata

Reply:
Removed